src/app/temp-converter/temp-converter.component.ts

```diff
@@ -3,13 +3,46 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-temp-converter',
   templateUrl: './temp-converter.component.html',
-  styleUrls: ['./temp-converter.component.css']
+  styleUrls: ['./temp-converter.component.css'],
 })
 export class TempConverterComponent implements OnInit {
+  constructor() {}
 
-  constructor() { }
-
-  ngOnInit(): void {
+  ngOnInit(): void {}
+  cel!: any;
+  cToF!: any;
+  feh!: any;
+  fToC!: any;
+  onInputCel() {
+    if (this.cel == 0) {
+      this.cToF = 0;
+    } else {
+      console.log(this.cel);
```

Comment:
Remove console.log from the final code. They are only used to debug your code while developing. These should not exist in the final running application as these may leak data to the end user.

Reply:
okay